Makefile

```diff
@@ -1,18 +1,66 @@
 NAME					:=	ircserv
 CC						:=	c++
-COMMON_FLAGS	:=	-Wall -Wextra -Werror \
+
+# Flags used by default
+CFLAGS				:=	-Wall -Wextra -Werror \
 									-std=c++14
 
+# Flags used by default, except with "make release"
+DFLAGS				:=	-DDEBUG \
+									-g3 \
+									-Wconversion \
+									-Wdouble-promotion \
+									-fsanitize=address,undefined,unreachable,null
```

Comment:
This needs to be seperate for lldb, for example a "make lldb" could be CFLAGS and DFLAGS without fsanitize

Reply:
There now exists make lldb which does not include -fsanitize flags